vitrina/api/migrations/0003_apikey_representative.py

```diff
@@ -0,0 +1,73 @@
+# Generated by Django 3.2.16 on 2022-11-03 07:16
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def create_representative_api_keys(apps, schema_editor):
+    ApiKey = apps.get_model('vitrina_api', 'ApiKey')
+    Representative = apps.get_model('vitrina_orgs', 'Representative')
+    Organization = apps.get_model('vitrina_orgs', 'Organization')
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    ct = ContentType.objects.get_for_model(Organization)
+
+    for key in ApiKey.objects.all():
+        if key.organization:
+            for rep in Representative.objects.filter(content_type=ct, object_id=key.organization.pk):
+                ApiKey.objects.create(
+                    version=key.version,
+                    api_key=key.api_key,
+                    enabled=key.enabled,
+                    expires=key.expires,
+                    representative=rep
+                )
+                rep.has_api_access = True
+                rep.save()
+
+
+def delete_non_representative_api_keys(apps, schema_editor):
+    ApiKey = apps.get_model('vitrina_api', 'ApiKey')
+
+    for key in ApiKey.objects.all():
+        if not key.representative:
+            key.delete()
```

Comment:
```suggestion
    ApiKey.objects.filter(representative__isnull=True).delete()
```

Electricity is expensive these days, so please, do not burn CPU if this is not needed.